CONTRIBUTING.md

```diff
@@ -2,58 +2,57 @@
 
 This page provides information about contributing code to the Jenkins core codebase.
 
-:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, check out [Participate].
+:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, here's a page on how to [Participate].
 
 ## Getting started
 
 1. Fork the repository on GitHub
 2. Clone the forked repository to your machine
-3. Install the development tools. In order to develop Jenkins, you need the following tools:
+3. Install the necessary development tools. In order to develop Jenkins, you need the following:
   * Java Development Kit (JDK) 8 or 11.
-    In Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
+    In the Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
     * For JDK 11 there might be some compatibility issues in developer tools,
       please see [this page](https://wiki.jenkins.io/display/JENKINS/Java+11+Developer+Guidelines#Java11DeveloperGuidelines-Knowndevelopertoolsissues) for more info.
       If you hit a new issue, please report it with a `java11-devtools-compatibility` label in our issue tracker.
-  * Maven 3.5.4 or above. You can [download maven].
+  * Maven 3.5.4 or above. You can [download maven] here.
   * Any IDE which supports importing Maven projects.
-  * Install [NodeJS](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found on the `war` module.
-    * Frontend tasks are ran using [yarn](https://yarnpkg.com/lang/en/). Run `npm install -g yarn` to install it.
+  * Install [NodeJS](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
+    * Frontend tasks are run using [yarn](https://yarnpkg.com/lang/en/). Run `npm install -g yarn` to install it.
 4. Setup your development environment as described in [Preparing for Plugin Development]
 
-If you want to contribute to Jenkins or just learn about the project,
+If you want to contribute to Jenkins, or just learn about the project,
 you can start by fixing some easier issues.
 In the Jenkins issue tracker we mark such issues as `newbie-friendly`.
-You can find them
-using this query for [newbie friendly issues].
+You can find them by using this query (check the link) for [newbie friendly issues].
 
 ## Building and Debugging
 
-The build flow for Jenkins core is built around Maven.
-There is a description of the [building and debugging process].
+The core of Jenkins' build flow is built around Maven.
```

Comment:
Introduces a semantic change (or an ambiguity, depending on how you read it): "Jenkins core" is a fixed term we use to refer to Jenkins without plugins.

Reply:
Done